homeassistant/components/tibber/config_flow.py

```diff
@@ -0,0 +1,67 @@
+"""Adds config flow for Tibber integration."""
+import asyncio
+import logging
+
+import aiohttp
+import tibber
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_ACCESS_TOKEN
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+
+from .const import DOMAIN  # pylint:disable=unused-import
+
+_LOGGER = logging.getLogger(__name__)
+
+DATA_SCHEMA = vol.Schema({vol.Required(CONF_ACCESS_TOKEN): str})
+
+
+class TibberConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for Tibber integration."""
+
+    VERSION = 1
+
```

Comment:
What is the connection class?

https://github.com/home-assistant/core/blob/752679c55d54c17683940a482fafb98a130c0556/homeassistant/components/hue/config_flow.py#L32